config.py

```diff
@@ -0,0 +1,8 @@
+HANDSTROKE_GAP = 1.0
```

Comment:
Could this just be a command line argument?

Reply:
Yes, but if we're going to make a GUI then it we'll want to be able to change settings without restarting the program.